portal/server/routes/login/index.js

```diff
@@ -79,7 +57,8 @@ router.post('/login', async (req, res) => {
       req.session.numberOfSendSignInLinkAttemptsRemaining = numberOfSendSignInLinkAttemptsRemaining;
     } catch (sendSignInLinkError) {
       if (sendSignInLinkError.response?.status === 403) {
-        return res.render('login/temporarily-suspended.njk');
+        req.session.numberOfSendSignInLinkAttemptsRemaining = -1;
+        return res.status(403).render('login/temporarily-suspended.njk');
```

Comment:
I've added this to return a 403 as there's existing precident for returning correct statuses on failed requests (ie delete account)